CMakeLists.txt

```diff
@@ -7,6 +7,8 @@ project( CloudCompareProjects )
 option(BUILD_TESTING "" OFF)
 include(CTest)
 
+include( GNUInstallDirs )
```

Comment:
Is it defined/existing on Windows?

Reply:
Yes, it exists on Windows. I just ran a test compilation with Windows 8, cmake 3.9, Visual Studio 2017.

Reply:
Since it's only used in the Linux case, please wrap in `if ( UNIX AND NOT APPLE )`.

Reply:
Thanks for the contribution. I’m very Happy to see a Debian contributor trying to fix issue upstream. Please take into account @asmaloney remark. else it LGTM.

Reply:
Per asmaloney's request, I moved the `include`.